File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ElectrodiagnosticTestsFormViewModel.cs
//$B8D220EA
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class ElectrodiagnosticTestsServiceController
    {
    }
}

namespace Core.Models
{
    public partial class ElectrodiagnosticTestsFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var electrodiagnosticTestsService = new ElectrodiagnosticTestsServiceController();
            electrodiagnosticTestsService.ElectrodiagnosticTestsFormInternal(this, objectContext);
            return (ElectrodiagnosticTests)objectContext.GetObject(this._ElectrodiagnosticTests.ObjectID, "ElectrodiagnosticTests");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ElectrodiagnosticTestsFormViewModel.ts
//$AE3476F5
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ElectrodiagnosticTests } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class ElectrodiagnosticTestsFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ElectrodiagnosticTestsFormViewModel, Core';
    @Type(() => ElectrodiagnosticTests)
    public _ElectrodiagnosticTests: ElectrodiagnosticTests = new ElectrodiagnosticTests();
}
